Makefile

```diff
@@ -51,6 +51,8 @@ IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.25.0
 
+CRDDESC_OVERRIDE ?= :maxDescLen=0
```

Comment:
Long term we might want to re-enable description on the `openstack-operator`, but good to have it to `0` for now (we did it in all the storage operators).